src/interfaces/featureWalkthrough.ts

```diff
@@ -0,0 +1,20 @@
+import { ReactFragment } from 'react'
+
+export interface Video {
+    mp4: string
+    webm: string
+}
+
+export interface VideoElement {
+    el: HTMLVideoElement | null
+    paused: boolean
+}
```

Comment:
These interfaces could maybe live in their own file since they aren't necessarily tied to `Features` interface. I'd like CPT to one day clean up our interfaces, move all our interfaces to their own files like this, I prefer `IFeature`, `I{Interface-Name}`, as a naming convention to make it clear it's an interface. But I realize this is a new initiative entirely 😬

Reply:
Thank you Tim! This is awesome, for the PR & the new issue to track our interface cleanup :)